core/src/main/java/org/infinispan/container/offheap/UnpooledOffHeapMemoryAllocator.java

```diff
@@ -18,7 +18,7 @@
    private static final Log log = LogFactory.getLog(UnpooledOffHeapMemoryAllocator.class, Log.class);
    private static final boolean trace = log.isTraceEnabled();
    private static final OffHeapMemory MEMORY = OffHeapMemory.INSTANCE;
-   private final AtomicLong amountAllocated = new AtomicLong();
+   private final LongAdder amountAllocated = new LongAdder();
```

Comment:
This is a good step, but I wonder if we could eliminate it completely when the data container is configured with MEMORY eviction.

Reply:
That is possible yes. I would say we can do that later though.